static/js/poll_modal.js

```diff
@@ -0,0 +1,70 @@
+import $ from "jquery";
+import {Sortable} from "sortablejs";
+
+import render_poll_modal_option from "../templates/poll_modal_option.hbs";
+
+export function update_option_delete_btn($container, display_flag) {
```

Comment:
Maybe a better name for this function would be `update_delete_btn_visibility`?

Reply:
agree, will update the name, thanks!